controller.go

```diff
@@ -147,14 +149,14 @@ func (c *healthCheckController) runServiceChecksByServiceNames(services []servic
 func (c *healthCheckController) runServiceChecksFor(categories map[string]category) ([]fthealth.CheckResult, map[string][]fthealth.CheckResult) {
 	categorisedResults := make(map[string][]fthealth.CheckResult)
 	serviceNames := getServiceNamesFromCategories(categories)
-	services := c.healthCheckService.getServicesByNames(serviceNames)
+	services := c.healthCheckService.getServicesMapByNames(serviceNames)
 	healthChecks := c.runServiceChecksByServiceNames(services, categories)
 
 	for catIndex, category := range categories {
 		if category.isSticky && category.isEnabled {
```

Comment:
I would rewrite this part the following way to remove one level of nesting.
if !(category.isSticky && category.isEnabled) {
continue
}
Maybe you can extract this part "category.isSticky && category.isEnabled" if you can name it in a meaningful way.